### ISSUE TITLE
Initial screen: Ctrl+L's Chat rightSidebar

### ISSUE THREAD

**Issue by danimesq** [cursor/cursor#884]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Just as [https://cursor.so](https://cursor.so)'s site has its own chat right at the begin, Cursor app's first screen should also have that "Ctrl+L Chat".
So, users can consult/ask questions before starting an new project/opening an existing one.
For unexisting projects, this can give users insights for prompting.

**Comment by leerob** [comment]
Thanks for opening this issue! We've moved our community support to the [Cursor Forum](https://forum.cursor.com) where you can get help from both the team and other users.

If this is a specific bug that you can still reproduce, please open a new issue with clear reproduction steps on the forum.

Thanks you!
